### PR TITLE
extended sdk-nestjs docs + cleanup

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
#### Description of the pull request (what is new / what has changed)
Extended NestJS SDK docs + rebranding + code cleanup

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+`doSomethingCpuIntensive() execution time: 20ms, CPU time 18ms`
+
+*Note that a big execution time does not necessarily have an impact on the CPU load of the application. That means that, for example, while waiting for an HTTP request, the JavaScript thread can process other things. That is not the case for CPU time. When a method consumes a lot of CPU time, Javascript will not be able to process other tasks, potentially causing a freeze until the CPU-intensive task is complete.*
+
+---
+
+[comment]: # (mx-context-auto)
+
+## Interceptors
+The package provides a series of [Nestjs Interceptors](https://docs.nestjs.com/interceptors) which will automatically log and set the CPU and overall duration for each request in a prom histogram ready to be scrapped by Prometheus.
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-monitoring.md:101]: `_prom_` or `Prometheus histogram`, plus a link to Prometheus?

File: docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+  publicApp.useGlobalInterceptors(...globalInterceptors);
+}
+```
+
+[comment]: # (mx-context-auto)
+
+## MetricsModule and MetricsService
+
+`MetricsModule` is a [Nestjs Module](https://docs.nestjs.com/modules) responsible for aggregating metrics data through `MetricsService` and exposing them to be consumed by Prometheus. `MetricsService` is extensible, you can define and aggregate your own metrics and expose them. By default it exposes a set of metrics created by the interceptors specified [here](#interceptors). Most of the Multiversx packages expose metrics by default through this service. For example [@multiversx/sdk-nestjs-redis](https://www.npmjs.com/package/@multiversx/sdk-nestjs-redis) automatically tracks the execution time of each redis query, overall redis health and much more, by leveraging the `MetricsService`.
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-monitoring.md:129]: "Most of the Multiversx packages" -> typo X.

**Review by andreibancioiu — APPROVED** [review]

**Review by tanghel — APPROVED** [review]

File: docs/integrators/accounts-management.md [hunk 3]
```diff
@@ -18,8 +18,8 @@ Accounts creation can be achieved through different approaches:
 - using the [MultiversX Web Wallet](https://wallet.multiversx.com/)
 - programmatically, using the [sdk-js - JavaScript SDK](/sdk-and-tools/sdk-js)
 - programmatically, using the [mxpy - Python SDK](/sdk-and-tools/sdk-py/)
-- programmatically, using the [erdgo - Golang SDK](/sdk-and-tools/erdgo)
-- programmatically, using the [erdjava - Java SDK](/sdk-and-tools/erdjava)
+- programmatically, using the [sdk-go - Golang SDK](/sdk-and-tools/sdk-go)
+- programmatically, using the [mxjava - Java SDK](/sdk-and-tools/mxjava)
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/integrators/accounts-management.md:22]: done

File: docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-cache.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+`sdk-nestjs-cache` is delivered via **npm** and it can be installed as follows:
+
+```
+npm install @multiversx/sdk-nestjs-cache
+```
+
+[comment]: # (mx-context-auto)
+
+## Utility
+The package exports **in memory cache service** and **remote cache service**.
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-cache.md:27]: done

File: docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-cache.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+
+[comment]: # (mx-context-auto)
+
+## Table of contents
+- [In Memory Cache](#in-memory-cache) - super fast in memory caching system based on [LRU cache](https://www.npmjs.com/package/lru-cache)
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-cache.md:32]: done

File: docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-cache.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+- [Redis Cache](#redis-cache) - Caching system based on [Redis](https://www.npmjs.com/package/@multiversx/sdk-nestjs-redis)
+- [Cache Service](#cache-service) - MultiversX caching system which combines in-memory and Redis cache, forming a two-layer caching system
+
+[comment]: # (mx-context-auto)
+
+## In memory cache
+In memory cache, available through `InMemoryCacheService`, is used to read and write data from and into the memory storage of your Node.js instance.
+
+*Note that if you have multiple instances of your application you must sync local cache across all your instances.*
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-cache.md:41]: done

File: docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-cache.md [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+  - `key`: The key of the item to update or create.
+  - `createValueFunc`: A function that creates the new value for the key.
+  - `ttl`: Time-to-live for the cached item in seconds.
+  - `cacheNullable` (optional): If set to `false`, the method will not cache null or undefined values. Default: `true`
+- **Returns:** A `Promise` that resolves to the updated or newly created value.
+
+[comment]: # (mx-context-auto)
+
+## Redis Cache
+Redis cache, available through `RedisCacheService`, is a caching system build ontop of Redis. It is used to share cache related information among multiple microservices.
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-cache.md:144]: done

File: docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-monitoring.md (hunk 1 above)
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-monitoring.md:101]: done

File: docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-monitoring.md (hunk 2 above)
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/sdk-nestjs/sdk-nestjs-monitoring.md:129]: done

**Review by andreibancioiu — APPROVED** [review]

**Review by tanghel — APPROVED** [review]

**Pull request merged**